.github/workflows/canary-integration-test.yml

```diff
@@ -309,11 +330,18 @@ jobs:
         name: pvc-db-wal
         path: test
 
-    - name: setup tmate session for debugging
-      if: failure()
+    - name: setup tmate session for debugging when event is PR
+      if: failure() && github.event_name == 'pull_request'
       uses: mxschmitt/action-tmate@v3
       timeout-minutes: 120
 
+    - name: setup tmate session for debugging when event is Push
```

Comment:
I would suggest having a single action. That action can have if/else condition.  Or multiple `if`s (in case `else` is not supported)

Having `setup tmate session for debugging when event is PR` and  `setup tmate session for debugging when event is Push` in every CI might be confusing.

Reply:
I'm not sure if multiple `if` are allowed but will check once the condition are working

Reply:
I agree but anyway we'll get only one debug session but yes I can change the name for these 2 jobs(if have some suggesions)